### PR TITLE
use coordsForChar when available

### PR DESCRIPTION
**Pull request by nightwing**
fixes https://github.com/replit/codemirror-vim/issues/131 

requires the new version of codemirror as discussed at https://discuss.codemirror.net/t/is-there-a-way-to-get-left-coordinate-of-a-character/6891

### CODE REVIEW

File: dev/index.ts [hunk 1]
```diff
@@ -44,6 +44,78 @@ function addOption(name, description?, onclick?) {
   return value
 }
 
+import {WidgetType} from "@codemirror/view"
```
Comment (sergeichestakov) [inline dev/index.ts:47]: can we add this to the existing import above?

File: src/block-cursor.ts [hunk 2]
```diff
@@ -170,6 +170,11 @@ function measureCursor(cm: CodeMirror, view: EditorView, cursor: SelectionRange,
     }
     let style = getComputedStyle(node as HTMLElement);
     let left = pos.left;
+    // TODO remove coordsAtPos when all supported versions of codemirror have coordsForChar api
```
Comment (sergeichestakov) [inline src/block-cursor.ts:173]: how far back are we planning on supporting?

**Review by sergeichestakov — APPROVED** [review]
neat! 📝

File: src/block-cursor.ts (hunk 2 above)
Comment (nightwing) [reply src/block-cursor.ts:173]: I am not sure. Maybe until we accumulate enough changes to bump the major version?

**Pull request merged**